ports/stm32/boards/common_isr_ram/common_isr.ld

```diff
@@ -14,7 +14,7 @@
        flash (in main.c) along with the isr_vector above.
     */
     . = ALIGN(4);
-    *(.text.pendsv_kbd_intr)
+    *(.text.mp_sched_keyboard_interrupt)
```

Comment:
@andrewleech it's curious that you only needed to put `pendsv_kbd_intr` in RAM.  That function calls `mp_sched_keyboard_interrupt` so I wonder why you didn't need to put the latter in RAM as well?

Reply:
Oh that is interesting... maybe neither of them actually need to be in ram... or more likely my testing didn't catch the scenario where ctrl-c was being handled by the UART isr at the exact moment flash was locked. Just before or just after it wouldn't matter. 
I think I added that function to the ram list based just on review of things called from the isr function but forgot to follow it down the call stack to subsequent functions.

Reply:
OK, thanks for confirming.  I'll leave this as it is in this PR because it makes sense to have this function in RAM.